zubhub_backend/zubhub/creators/views.py

```diff
@@ -134,7 +134,7 @@ def get_serializer_class(self):
             return CreatorMinimalSerializer
         else:
             return CreatorSerializer
-        
```

Comment:
Hello @Mutombe , thank you for working on this. You added extra space to this line. Can you remove it since it is not part of this PR?